.teamcity/pom.xml

```diff
@@ -1,15 +1,9 @@
 <?xml version="1.0"?>
 <project>
   <modelVersion>4.0.0</modelVersion>
-  <name>Gradle Guides Plugin</name>
-  <properties>
-    <kotlin.version>1.3.61</kotlin.version>
-    <teamcity.dsl.version>2021.1</teamcity.dsl.version>
-    <mockk.version>1.9</mockk.version>
```

Comment:
😲

Reply:
None of the explicit versions are needed, and they are probably horribly outdated or even incompatible

Reply:
They automatically come from the TeamCity version from the settings.kts

Reply:
Yeah, I was suprised they were here in the first place